### DISCUSSION TITLE
Authentication with proof

### DISCUSSION THREAD

**Discussion by pourya-azad** [community/community#66065] [GitHub Education]
### Hi everyone,

I tried several times to get the Github student pack, but every time it was rejected for simple reasons, for example, once because the university logo was not in the photo / once because the photo of my student card was not translated into English!  / And when I translated it because the photo was edited / And now, no matter how much I try to upload my degree and click on approval, I return to the same page / I really don't know what else to do to approve my degree

**Comment by Gatix1** [comment]
I think there is only a way, to make some copy with translation and show it too in the picture with the original
